src/common/hooks/api/category.js

```diff
@@ -1,14 +1,16 @@
 import { useQuery } from 'react-query';
-import { CategoryService } from 'network';
+import { CategoryService } from '@network';
 
 export const CATEGORY_URL = '/category';
 export const CATEGORIES_URL = '/categories';
 
 export const getCategory = () => {
-  const { isError, data } = useQuery(
-    [CATEGORIES_URL],
-    CategoryService.getCategories,
-  );
+  const { isError, data } = useQuery([CATEGORIES_URL], CategoryService.getCategories);
 
   return { isError, categories: data?.data ?? [] };
```

Comment:
```suggestion
  const { isError, data = {data: []} } = useQuery([CATEGORIES_URL], CategoryService.getCategories);

  return { isError, categories: data.data };
```

마찬가지로 옵셔널체이닝과 널리시 연산자를 조합하기보단 초기값을 지정해주면 깔끔할것같아요